packages/component-submission/server/resolvers/submitSurveyResponse.js

```diff
@@ -0,0 +1,28 @@
+const { submitSurveyUseCase } = require('../use-cases')
+const SurveyResponse = require('@elifesciences/component-model-survey-response')
+  .model
+const config = require('config')
+
+async function submitSurveyResponse(_, { data }, { user: userId }) {
+  const { surveyId, submissionId, answers } = data
+
+  try {
+    return await submitSurveyUseCase(
+      { SurveyResponse },
+      {
+        surveyId,
+        submissionId,
+        answers,
```

Comment:
Does this contain the question string to? if so we should name it `questionsAndAnswers` or something along those lines

Reply:
I think we resolved this face-to-face, it's defined in the GQL where the answer is defined as containing a question - let's improve this later on outside the scope of this PR.